libpgmodeler_ui/src/modelwidget.cpp

```diff
@@ -1004,19 +1004,14 @@ void ModelWidget::handleObjectModification(BaseGraphicObject *object)
 
 void ModelWidget::emitSceneInteracted(void)
 {
-	if(selected_objects.empty())
+	if(scene->selectedItems().empty())
```

Comment:
Hmm.. I'm worried about this... because ```selected_objects``` is not the same as ```scene->selectedItems()```. I'll investigate this carefully.

Reply:
Spot on, I realized yesterday that I had not checked very carefully the side effects on this, there may be...
The point of it is to allow a column selected to appear correctly in the scene info widget.
The current selected_objects is a BaseGraphicObject, and as you know a TableObject is not a subclass of it. But in libobjrenderer, TableObjectView inherits BaseObjectView.